<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SuperCalculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>827</width>
      <height>100111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="input_label">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Input:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_field">
       <property name="minimumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="var_table">
       <property name="minimumSize">
        <size>
         <width>724</width>
         <height>428</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>724</width>
         <height>428</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Variable</string>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="result_label">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Result:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="result_field">
       <property name="minimumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="RPN_label">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RPN:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="RPN_field">
       <property name="minimumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>724</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>575</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="lock_check">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>415</y>
      <width>60</width>
      <height>53</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>53</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lock</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
